strings/names/dogs.txt

```diff
@@ -0,0 +1,30 @@
+
+Sure, here's an extended list of dog names formatted similarly to the provided list of cat names:
```

Comment:
pfff probably best to remove the chatgpt comment here
```suggestion
```

Reply:
Bahaha that's funny my bad